DNN Platform/Tests/DotNetNuke.Tests.Web/InternalServices/SearchServiceControllerTests.cs

```diff
@@ -145,7 +147,15 @@ public void SetUp()
             CBO.SetTestableInstance(_mockCBO.Object);
 
             //create instance of the SearchServiceController
-            _searchServiceController = new SearchServiceController(HtmlModDefId);
+            var request = new HttpRequestMessage();
+            var configuration = new HttpConfiguration();
+            var provider = new Mock<ITabAndModuleInfoProvider>();
+            ModuleInfo expectedModule;
+            provider.Setup(x => x.TryFindModuleInfo(request, out expectedModule)).Returns(true);
+            configuration.AddTabAndModuleInfoProvider(provider.Object);
+            request.Properties[HttpPropertyKeys.HttpConfigurationKey] = configuration;
+            _searchServiceController = new SearchServiceController(HtmlModDefId){Request = request };
```

Comment:
It is a bit weird to set the request message as property...

Reply:
get the code from: https://github.com/dnnsoftware/Dnn.Platform/blob/development/DNN%20Platform/Tests/DotNetNuke.Tests.Web/Api/DnnApiControllerTests.cs#L58

Reply:
Mhm. I does work, all right. I give you that :+1: